CrewChiefV4/App.config

```diff
@@ -100,14 +100,11 @@
       <setting name="custom_device_guid" serializeAs="String">
         <value />
       </setting>
-      <setting name="r3e_launch_params" serializeAs="String">
```

Comment:
my suggestion here is to rename these to '..._launch_params' for now, and reinstate the old help text (and anything else that was changed). We could do a separate param for steam app ID if we need to later on - keep this pull request's scope to AMS / rF1 fixes only if we can